frontend/appflowy_flutter/lib/workspace/presentation/settings/widgets/settings_appearance/color_scheme_setting.dart

```diff
@@ -107,7 +116,11 @@ class ColorSchemeUploadPopover extends StatelessWidget {
                   children: [
                     ...AppTheme.builtins
                         .map(
-                          (theme) => _themeItemButton(context, theme.themeName),
+                          (theme) => _themeItemButton(
+                            context,
+                            controller,
```

Comment:
It probably isn't necessary to pass the controller here since you're in the same scope